config/config.apps.sample.php

```diff
@@ -269,11 +269,12 @@
  * login without requiring the user to click a button. The default is `false`.
  *
  * auto-provision::
- * If auto-provision is setup, an ownCloud user will be created if not exists, after successful
+ * If `auto-provision` is setup, an ownCloud user will be created if not exists, after successful
  * login using openid connect. The config parameters `mode` and `search-attribute` will be used
  * to create a unique user so that the lookup mechanism can find the user again. This is where
- * an LDAP setup is usually required.
- * If auto-provision is not setup or required, it is expected that the user exists and you
+ * an LDAP setup is usually required. The profile picture will only be transferred upon account
+ * creation, but will not get updated afterwards like when it get changed.
```

Comment:
```suggestion
 * creation, but will not be updated afterwards if it changes in the connected IdP.
```

Not quite sure what to call the "upstream identity provider/authentication source.